docs/how-to-install.md

```diff
@@ -6,145 +6,170 @@ license: "This software is licensed under the Apache License version 2."
-    |   └── jdbc-doris/                         # JDBC doris catalog dependencies and configurations.
-    |   └── jdbc-mysql/                         # JDBC MySQL catalog dependencies and configurations.
-    |   └── jdbc-postgresql/                    # JDBC PostgreSQL catalog dependencies and configurations.
-    |   └── kafka/                              # Apache Kafka PostgreSQL catalog dependencies and configurations.
-    |   └── lakehouse-iceberg/                  # Apache Iceberg catalog dependencies and configurations.
-    |   └── lakehouse-paimon/                   # Apache Paimon catalog dependencies and configurations.
-    |── conf/                                   # All configurations for Gravitino.
-    |   ├── gravitino.conf                      # Gravitino server and Gravitino Iceberg REST server configuration.
-    |   ├── gravitino-iceberg-rest-server.conf  # Gravitino server configuration.
-    |   ├── gravitino-env.sh                    # Environment variables, etc., JAVA_HOME, GRAVITINO_HOME, and more.
-    |   └── log4j2.properties                   # log4j configuration for the Gravitino server and Gravitino Iceberg REST server.
-    |── libs/                                   # Gravitino server dependencies libraries.
-    |── logs/                                   # Gravitino server and Gravitino Iceberg REST server logs. Automatically created after the server starts.
-    |── data/                                   # Default directory for the Gravitino server to store data.
-    |── iceberg-rest-server/                    # Gravitino Iceberg REST server package and dependencies libraries.
-    └── scripts/                                # Extra scripts for Gravitino.
-```
+### Get the Gravitino package
+
+The Gravitino package comprises the Gravitino server and the Gravitino Iceberg REST server.
+You can manage these servers independently or run them concurrently on a single server.
+
+1. Download the latest Gravitino package from [GitHub](https://github.com/apache/gravitino/releases).
+
+   :::note
+   You can also build Gravitino by following the instructions in [How to Build Gravitino](./how-to-build.md).
+
+   - If you build Gravitino using the `./gradlew compileDistribution` command, you can find the Gravitino package in the `distribution/package` directory.
 
-#### Initialize the RDBMS (Optional)
+   - If you build Gravitino using the `./gradlew assembleDistribution` command, you can get the compressed Gravitino package with the name `gravitino-<version>-bin.tar.gz` in the `distribution` directory with sha256 checksum file `gravitino-<version>-bin.tar.gz.sha256`.
+   :::
 
-If you want to use the relational backend storage, you need to initialize the RDBMS first. For the details on initializing the RDBMS, please check [How to use relational backend storage](./how-to-use-relational-backend-storage.md).
+2. Extract the Gravitino package and you should have these directories and files:
```

Comment:
Perhaps this is better? "After extracting the Gravitino package, you should see the following directories and files:"

Reply:
From my understanding, after.... is a result of an action not a step. here I use the word and to show the sequence of two actions.

Reply:
Fine to keep as is.